src/SSW.SophieBot/SSWSophieBot/settings/botsettings.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
The botsettings.json is created manually. We should include it in source as it is used for configuring custom dialogs, see below. 
![image](https://user-images.githubusercontent.com/16027480/129533237-3edd145d-a249-41c1-bfde-826b24df73c2.png)
Since we are excluding appsettings.json for security purposes, I created botsettings.json to place the public bot configurations.

Reply:
Approve for now, as per conversation, we need to improve the botsetting and app setting